_opensearch/rest-api/index-apis/put-mapping.md

```diff
@@ -36,23 +39,29 @@ PUT /<target-index>/_mapping
 PUT /_mapping
 ```
 
+You can also use the put mapping operation to update multiple indexes with one request.
+
+```
+PUT /<target-index1>,<target-index2>/_mapping
+```
+
 ## URL parameters
 
-All put mapping parameters are optional.
+All update mapping parameters are optional.
 
 Parameter | Data Type | Description
 :--- | :--- | :---
-&lt;target-index&gt; | Data Type | The index to add the mapping to. If you do not specify this parameter, OpenSearch adds the mapping to all indices within the cluster.
-allow_no_indices | Boolean | Whether to ignore wildcards that don’t match any indices. Default is `true`.
-expand_wildcards | String | Expands wildcard expressions to different indices. Combine multiple values with commas. Available values are `all` (match all indices), `open` (match open indices), `closed` (match closed indices), `hidden` (match hidden indices), and `none` (do not accept wildcard expressions), which must be used with `open`, `closed`, or both. Default is `open`.
-ignore_unavailable | Boolean | If true, OpenSearch does not include missing or closed indices in the response.
+&lt;target-index&gt; | N/A | Specifies an index to associate the mapping. If you do not specify this parameter, OpenSearch adds the mapping to all indexes within the cluster.
+allow_no_indices | Boolean | Whether to ignore wildcards that don’t match any indexes. Default is `true`.
+expand_wildcards | String | Expands wildcard expressions to different indexes. Combine multiple values with commas. Available values are `all` (match all indexes), `open` (match open indexes), `closed` (match closed indexes), `hidden` (match hidden indexes), and `none` (do not accept wildcard expressions), which must be used with `open`, `closed`, or both. Default is `open`.
+ignore_unavailable | Boolean | If true, OpenSearch does not include missing or closed indexes in the response.
 master_timeout | Time | How long to wait for a connection to the master node. Default is `30s`.
-timeout | Time | How long to wait for a response from the cluster. Default is `30s`.
-write_index_only | Boolean | Whether OpenSearch should add the mapping only to write indexes. If false, OpenSearch can add the mapping to all indexes with the same alias. See [alias]({{site.url}}{{site.baseurl}}/opensearch/rest-api/alias/#request-body) for more information. Default is false.
+timeout | Time | How long to wait for the response to return. Default is `30s`.
+write_index_only | Boolean | Whether OpenSearch should apply mapping updates only to the write index.
 
 ## Request body
 
-The request body must contain the `properties` object, which has all of the mappings that you want to add.
+The request body must contain `properties`, which has all of the mappings that you want to update.
```

Comment:
"...which has all of the mappings you want to create or update."

Reply:
good catch thanks